models.py

```diff
@@ -87,9 +87,10 @@ class User(db.Model, UserMixin):
     photo = db.Column(db.String())
     photo_type = db.Column(db.String(255))
 
-    def __init__(self, **kwargs):
+    def __init__(self, email, **kwargs):
```

Comment:
this change should be removed, it will break things that rely on the specific function signature

Reply:
taken out the email part